atest/testdata/running/test_template.robot

```diff
@@ -169,7 +169,7 @@ Templates, timeouts, and for loops
     \    0.05 seconds
 
 Templates and syntax errors
-    [Documentation]  FAIL  Variable '\${nonexisting}' not found.
+    [Documentation]  FAIL Several failures occurred:\n\n 1) Variable '${nonexisting}' not found.\n\n 2) not compared != anymore
     [Template]  Should be equal
     ${nonexisting}      something
     not compared  anymore
```

Comment:
The idea of this test is checking that execution of a templated test stops if there is a syntax error. Now that non-existing variable isn't a syntax error anymore, the test doesn't make much sense in this format. It should be updated to use something that truly is a syntax error. It probably would be easiest to use the template keyword with wrong number of arguments:

```robotframework
*** Test Cases ***
Templates and syntax errors
    [Documentation]    FAIL xxx
    [Template]    Should be equal
    we    have    way    too    many    arguments    here
    not compared    anymore
```

It would actually be a good idea to add a new test here verifying that the whole execution isn't stopped if a variable doesn't exist.